_layouts/getstarted.html

```diff
@@ -9,6 +9,23 @@ <h1 class="text-caps">{{page.title}} {{page.docversion}}</h1>
     <div class="grid__item width-12-12 width-12-12-mobile">
         <div>
             {{ content }}
+            {% comment %}
+            {% This should be part of the content in geststarted.adoc %}
+            {% But since we want to use the variable for the latest WildFly version %}
+            {% to render the link to the OS guide, this is part of the layout %}
+            {% endcomment %}
+            {% assign latestRelease = site.data.releases[site.current_release_index] %}
+            <div class="sect1">
+                <h2 id="next-steps">Next Steps</h2>
+                <div class="sectionbody">
+                    <div class="paragraph">
+                        <p>
+                            To learn more about WildFly, you can read its <a href="https://docs.wildfly.org" target="_blank" rel="noopener">documentation</a>.<br>
+                            If you want to learn how to use WildFly on OpenShift, read the <a href="https://docs.wildfly.org/{{latestRelease.version}}/Getting_Started_on_OpenShift.html" target="_blank" rel="noopener">Getting Started with WildFly on OpenShift Guide</a>
```

Comment:
This produces a broken URL, as latestRelease.version is 31.0.0.Final but the docs URL is segment is "31".

You need to add another field in _data/releases.yaml.

Reply:
Good catch!